src/components/leaderboards/View.vue

```diff
@@ -202,10 +202,6 @@ async function revealRestore() {
 	background: crimson;
 }
 
-.table-header {
-	font-weight: bold;
-}
-
```

Comment:
Why this change?

Reply:
Eh

If there was a reason I've forgotten. I'll revert.